packages/lesswrong/components/events/modules/HighlightedEventCard.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { registerComponent } from '../../../lib/vulcan-lib';
+import { Link } from '../../../lib/reactRouterWrapper';
+import { createStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { prettyEventDateTimes } from '../../../lib/collections/posts/helpers';
+import { useTimezone } from '../../common/withTimezone';
+import { cloudinaryCloudNameSetting } from '../../../lib/publicSettings';
+import { useTracking } from '../../../lib/analyticsEvents';
+
+// space pic for events with no img
+export const DEFAULT_EVENT_IMG = 'Banner/yeldubyolqpl3vqqy0m6'
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    maxWidth: 800,
+    height: 350,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center',
+    background: theme.palette.primary.main,
+    textAlign: 'center',
+    color: 'white',
+    borderRadius: 0,
+    margin: 'auto',
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: -4,
+      marginRight: -4,
+    }
+  },
+  content: {
+    overflow: 'visible'
+  },
+  spinnerContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '100%'
+  },
+  spinner: {
+    color: 'white'
+  },
+  row: {
+    marginTop: 8
+  },
+  title: {
+    ...theme.typography.headline,
+    display: 'inline',
+    background: 'black',
```

Comment:
I thought about doing that since I do think it looks better, but I was worried about contrast when there is white in the background img. I tried it out, seems ok to me:

<img width="664" alt="Screen Shot 2022-01-27 at 11 50 13 PM" src="https://user-images.githubusercontent.com/9057804/151493832-6bcf82d6-aaed-4eed-ad9b-1992ab7e282f.png">